libs/ui-components/src/lib/Footer/Socials.tsx

```diff
@@ -0,0 +1,56 @@
+import { Button, Flex } from 'antd';
+
+const WebSvg = () => (
```

Comment:
please export these svgs to .svg files

Reply:
I wanted to do it this way first, but there are problems with making them visible in apps... looks like we need to add a particular setup to each app's project.json https://nx.dev/recipes/tips-n-tricks/include-assets-in-build (easy to forget with many apps) or create a custom builder that will use this assets config, or even add some script that will copy all the assets https://stackoverflow.com/questions/73343438/nx-shared-asset-library 
all those I found more complicated than just having svgs as component, at least for now

Reply:
there's a way to have a global setup actually (so no need to copy to each app), but this will copy all the assets from the lib to the app regardless of whether it uses new footer or not 

anyway, lmkyt @truemiller @mohandast52

Reply:
can we not just add them to a shared lib called "images"?
i don't have too much nx experience

Reply:
we can, but we'll have the same problem with configs - the images need to be added to each app somehow, so they are accessible from it. the only way to do it is to add a special config which will copy all of them to each app images folder